x-pack/test/security_solution_cypress/cypress/screens/ai_assistant.ts

```diff
@@ -43,7 +43,8 @@ export const QUICK_PROMPT_BODY_INPUT = '[data-test-subj="quick-prompt-prompt"]';
 export const SEND_TO_TIMELINE_BUTTON = '[data-test-subj="sendToTimelineEmptyButton"]';
 export const SHOW_ANONYMIZED_BUTTON = '[data-test-subj="showAnonymizedValues"]';
 export const SUBMIT_CHAT = '[data-test-subj="submit-chat"]';
-export const SYSTEM_PROMPT = '[data-test-subj="systemPromptText"]';
+export const SYSTEM_PROMPT = '[data-test-subj="promptSuperSelect"]';
+export const PROMPT_SELECT = '[data-test-subj="promptSuperSelect"]';
```

Comment:
nit: It appears the only references to both `SYSTEM_PROMPT` and `PROMPT_SELECT` are in the same file, `x-pack/test/security_solution_cypress/cypress/tasks/assistant.ts`, so it may be possible to combine them